actions-server/src/handlers/joinGame.ts

```diff
@@ -36,14 +37,16 @@ const handler = async (req: Request, res: Response) => {
         })
         if (data_insert?.insert_players_one) {
           playerId = data_insert.insert_players_one.id
+          hostExists = Boolean(data_insert.insert_players_one.game.host_id)
```

Comment:
this line is likely not needed, not sure of a case where you could join a game and there is no host (because the first person to join any game becomes the host!)